base/src/main/java/io/spine/util/NamedProperty.java

```diff
@@ -31,10 +31,9 @@
  * The abstract base for classes obtaining a value of a named property.
  *
  * @param <T> the type of the property
- * @param <O> the type of the object from which we get the value
- * @author Alexander Yevsyukov
+ * @param <H> the type of the object from which we get the value
  */
-public abstract class NamedProperty<T, O> {
+public abstract class NamedProperty<T, H> {
```

Comment:
I assume `H` stands for "holder"?

To me it looks more `S` (source) or `O` (object, as previously).

Reply:
We're recently renaming `<O>` to something less looking like zero.